modules/tinymce/src/core/main/ts/api/util/VK.ts

```diff
@@ -22,9 +22,11 @@ interface VK {
   PAGE_DOWN: number;
   END: number;
   HOME: number;
+  CTRL: number;
+  META: number;
 
-  modifierPressed: (e: KeyboardLikeEvent) => boolean;
-  metaKeyPressed: (e: KeyboardLikeEvent) => boolean;
+  modifierPressed: (e: (KeyboardLikeEvent & { keyCode?: number })) => boolean;
+  metaKeyPressed: (e: (KeyboardLikeEvent & { keyCode?: number })) => boolean;
```

Comment:
This is should be part of the `KeyboardLikeEvent` instead of patched here.

Reply:
I thought putting it inside `KeyboardLikeEvent` breaks some other code. I was mistaken. Will fix now!

Reply:
Done!